src/main/java/minegame159/meteorclient/gui/screens/NotebotHelpScreen.java

```diff
@@ -0,0 +1,58 @@
+package minegame159.meteorclient.gui.screens;
+
+import minegame159.meteorclient.MeteorClient;
+import minegame159.meteorclient.gui.GuiTheme;
+import minegame159.meteorclient.gui.WindowScreen;
+import minegame159.meteorclient.gui.widgets.containers.WTable;
+
+public class NotebotHelpScreen extends WindowScreen {
+
+
+    public NotebotHelpScreen(GuiTheme theme) {
+        super(theme, "Notebot - Help");
+
+        WTable list = add(theme.table()).expandCellX().widget();
```

Comment:
You don't need a table, the WindowScreen itself is a vertical list.

Reply:
ok